### ISSUE TITLE
feat: add 'transparent' to tailwind color list

### ISSUE THREAD

**Issue by crotru** [Decathlon/vitamin-web#851]
**Feature request**

`transparent` should be in the color presets.
This would allow us to use classes like `vtmn-border-transparent`, `vtmn-bg-transparent`
Those are actually useful to "cancel" properties in a mobile-first approach :
```html
<div class="vtmn-bg-background-brand-primary desktop:vtmn-bg-transparent">
  <!-- ... -->
</div>

<div class="vtmn-border-border-primary desktop:vtmn-border-transparent">
  <!-- ... -->
</div>

```

**Preferred solution**

This can be done by adding `'transparent'` value in the [color presets](https://github.com/Decathlon/vitamin-web/blob/main/packages/sources/css/presets/tailwind/tailwind-preset.js)

**Alternatives**

It can be extended in local config of projects using `@vtmn/css-tailwind-preset` but I feel this seems like a good add-on to "core" vitamin

**Related code**

in the [tailwind preset file](https://github.com/Decathlon/vitamin-web/blob/main/packages/sources/css/presets/tailwind/tailwind-preset.js) : 

```
module.exports = {
  // ...

  theme: {
    // ...

    colors: {

      transparent: 'transparent',

      // ...
      // ...
      // ...

```

**Comment by lauthieb** [comment]
Thansk @crotru for your proposal. I'll do it.

**Comment by lauthieb** [comment]
Hi @crotru,
This is done thanks to this PR https://github.com/Decathlon/vitamin-web/pull/879
It's now available in `@vtmn/css-tailwind-preset>=0.33.31` 
Thanks again ;)

**Comment by crotru** [comment]
Thank you :)
